# TAP results for REGR
not ok 1 - Result of: test_Aussie_Specialist_Club (__main__.REGR) - 
# Traceback (most recent call last):
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 58, in testPartExecutor
#     yield
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 600, in run
#     testMethod()
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 798, in test_Aussie_Specialist_Club
#     verf('#nav-main-panel-5 a[href*="aussie-specialist-club/aussie-specialist-photos.html"]')
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 142, in assertElementPresent
#     waitfor.visibility_of(self.driver.find_element_by_css_selector(selector))))
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 434, in find_element_by_css_selector
#     return self.find_element(by=By.CSS_SELECTOR, value=css_selector)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 744, in find_element
#     {'using': by, 'value': value})['value']
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 233, in execute
#     self.error_handler.check_response(response)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 194, in check_response
#     raise exception_class(message, screen, stacktrace)
# selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: {"method":"css selector","selector":"#nav-main-panel-5 a[href*=\"aussie-specialist-club/aussie-specialist-photos.html\"]"}
# Stacktrace:
#     at FirefoxDriver.prototype.findElementInternal_ (file:///C:/Users/bzalakos/AppData/Local/Temp/tmpi8soxyft/extensions/fxdriver@googlecode.com/components/driver-component.js:10770)
#     at fxdriver.Timer.prototype.setTimeout/<.notify (file:///C:/Users/bzalakos/AppData/Local/Temp/tmpi8soxyft/extensions/fxdriver@googlecode.com/components/driver-component.js:625)
not ok 2 - Result of: test_Aussie_Specialist_Photos (__main__.REGR) - 
# Traceback (most recent call last):
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 58, in testPartExecutor
#     yield
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 600, in run
#     testMethod()
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 829, in test_Aussie_Specialist_Photos
#     elec('#nav-main-panel-5 a[href*="aussie-specialist-photos.html"]').click()
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 68, in reul
#     elle = fiel(*parg, **karg)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 434, in find_element_by_css_selector
#     return self.find_element(by=By.CSS_SELECTOR, value=css_selector)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 744, in find_element
#     {'using': by, 'value': value})['value']
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 233, in execute
#     self.error_handler.check_response(response)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 194, in check_response
#     raise exception_class(message, screen, stacktrace)
# selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: {"method":"css selector","selector":"#nav-main-panel-5 a[href*=\"aussie-specialist-photos.html\"]"}
# Stacktrace:
#     at FirefoxDriver.prototype.findElementInternal_ (file:///C:/Users/bzalakos/AppData/Local/Temp/tmpbl42hbul/extensions/fxdriver@googlecode.com/components/driver-component.js:10770)
#     at fxdriver.Timer.prototype.setTimeout/<.notify (file:///C:/Users/bzalakos/AppData/Local/Temp/tmpbl42hbul/extensions/fxdriver@googlecode.com/components/driver-component.js:625)
not ok 3 - Result of: test_Aussie_Store (__main__.REGR) - 
# Traceback (most recent call last):
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 58, in testPartExecutor
#     yield
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 600, in run
#     testMethod()
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 972, in test_Aussie_Store
#     phone = elec('[name="countrycode"]').get_attribute('value') +\
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 68, in reul
#     elle = fiel(*parg, **karg)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 434, in find_element_by_css_selector
#     return self.find_element(by=By.CSS_SELECTOR, value=css_selector)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 744, in find_element
#     {'using': by, 'value': value})['value']
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 233, in execute
#     self.error_handler.check_response(response)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 194, in check_response
#     raise exception_class(message, screen, stacktrace)
# selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: {"method":"css selector","selector":"[name=\"countrycode\"]"}
# Stacktrace:
#     at FirefoxDriver.prototype.findElementInternal_ (file:///C:/Users/bzalakos/AppData/Local/Temp/tmpeftudwsc/extensions/fxdriver@googlecode.com/components/driver-component.js:10770)
#     at fxdriver.Timer.prototype.setTimeout/<.notify (file:///C:/Users/bzalakos/AppData/Local/Temp/tmpeftudwsc/extensions/fxdriver@googlecode.com/components/driver-component.js:625)
not ok 4 - Result of: test_Campaign (__main__.REGR) - 
# Traceback (most recent call last):
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 58, in testPartExecutor
#     yield
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 600, in run
#     testMethod()
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 867, in test_Campaign
#     raise NotImplementedError("For now, go check it yourself?")
# NotImplementedError: For now, go check it yourself?
ok 5 - Result of: test_Change_Password (__main__.REGR) - 
not ok 6 - Result of: test_Contact_Us (__main__.REGR) - Checks the Contact Us page.
# Traceback (most recent call last):
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 58, in testPartExecutor
#     yield
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 600, in run
#     testMethod()
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 470, in test_Contact_Us
#     elec('a[href*="contact-us.html"]').click()
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webelement.py", line 74, in click
#     self._execute(Command.CLICK_ELEMENT)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webelement.py", line 457, in _execute
#     return self._parent.execute(command, params)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 233, in execute
#     self.error_handler.check_response(response)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 194, in check_response
#     raise exception_class(message, screen, stacktrace)
# selenium.common.exceptions.ElementNotVisibleException: Message: Element is not currently visible and so may not be interacted with
# Stacktrace:
#     at fxdriver.preconditions.visible (file:///C:/Users/bzalakos/AppData/Local/Temp/tmp98a224wy/extensions/fxdriver@googlecode.com/components/command-processor.js:10092)
#     at DelayedCommand.prototype.checkPreconditions_ (file:///C:/Users/bzalakos/AppData/Local/Temp/tmp98a224wy/extensions/fxdriver@googlecode.com/components/command-processor.js:12644)
#     at DelayedCommand.prototype.executeInternal_/h (file:///C:/Users/bzalakos/AppData/Local/Temp/tmp98a224wy/extensions/fxdriver@googlecode.com/components/command-processor.js:12661)
#     at fxdriver.Timer.prototype.setTimeout/<.notify (file:///C:/Users/bzalakos/AppData/Local/Temp/tmp98a224wy/extensions/fxdriver@googlecode.com/components/command-processor.js:625)
not ok 7 - Result of: test_Download_Qualification_Badge (__main__.REGR) - 
# Traceback (most recent call last):
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 58, in testPartExecutor
#     yield
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 600, in run
#     testMethod()
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 848, in test_Download_Qualification_Badge
#     elec('#nav-main-panel-5 a[href*="aussie-specialist-club/asp-logo.html"]').click()
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 68, in reul
#     elle = fiel(*parg, **karg)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 434, in find_element_by_css_selector
#     return self.find_element(by=By.CSS_SELECTOR, value=css_selector)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 744, in find_element
#     {'using': by, 'value': value})['value']
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 233, in execute
#     self.error_handler.check_response(response)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 194, in check_response
#     raise exception_class(message, screen, stacktrace)
# selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: {"method":"css selector","selector":"#nav-main-panel-5 a[href*=\"aussie-specialist-club/asp-logo.html\"]"}
# Stacktrace:
#     at FirefoxDriver.prototype.findElementInternal_ (file:///C:/Users/bzalakos/AppData/Local/Temp/tmp1if_gmtv/extensions/fxdriver@googlecode.com/components/driver-component.js:10770)
#     at fxdriver.Timer.prototype.setTimeout/<.notify (file:///C:/Users/bzalakos/AppData/Local/Temp/tmp1if_gmtv/extensions/fxdriver@googlecode.com/components/driver-component.js:625)
ok 8 - Result of: test_Famils (__main__.REGR) - 
not ok 9 - Result of: test_Favourites (__main__.REGR) - 
# Traceback (most recent call last):
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 58, in testPartExecutor
#     yield
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 600, in run
#     testMethod()
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 659, in test_Favourites
#     mtc = int(elec('.my-trip-count').text) or 0
# ValueError: invalid literal for int() with base 10: ''
not ok 10 - Result of: test_Filtered_Search__Fact_Sheets (__main__.REGR) - Tests the Fact Sheet Search.
# Traceback (most recent call last):
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 58, in testPartExecutor
#     yield
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 600, in run
#     testMethod()
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 341, in test_Filtered_Search__Fact_Sheets
#     elec('#nav-main-panel-2 a[href*="fact-sheets-overview.html"]').click()
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webelement.py", line 74, in click
#     self._execute(Command.CLICK_ELEMENT)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webelement.py", line 457, in _execute
#     return self._parent.execute(command, params)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 233, in execute
#     self.error_handler.check_response(response)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 194, in check_response
#     raise exception_class(message, screen, stacktrace)
# selenium.common.exceptions.ElementNotVisibleException: Message: Element is not currently visible and so may not be interacted with
# Stacktrace:
#     at fxdriver.preconditions.visible (file:///C:/Users/bzalakos/AppData/Local/Temp/tmpg4no593p/extensions/fxdriver@googlecode.com/components/command-processor.js:10092)
#     at DelayedCommand.prototype.checkPreconditions_ (file:///C:/Users/bzalakos/AppData/Local/Temp/tmpg4no593p/extensions/fxdriver@googlecode.com/components/command-processor.js:12644)
#     at DelayedCommand.prototype.executeInternal_/h (file:///C:/Users/bzalakos/AppData/Local/Temp/tmpg4no593p/extensions/fxdriver@googlecode.com/components/command-processor.js:12661)
#     at fxdriver.Timer.prototype.setTimeout/<.notify (file:///C:/Users/bzalakos/AppData/Local/Temp/tmpg4no593p/extensions/fxdriver@googlecode.com/components/command-processor.js:625)
not ok 11 - Result of: test_Filtered_Search__Itineraries (__main__.REGR) - Checks the Itineraries Search.
# Traceback (most recent call last):
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 58, in testPartExecutor
#     yield
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 600, in run
#     testMethod()
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 328, in test_Filtered_Search__Itineraries
#     elec('#nav-main-panel-2 a[href*="itineraries-search-and-feature.html"]').click()
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webelement.py", line 74, in click
#     self._execute(Command.CLICK_ELEMENT)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webelement.py", line 457, in _execute
#     return self._parent.execute(command, params)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 233, in execute
#     self.error_handler.check_response(response)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 194, in check_response
#     raise exception_class(message, screen, stacktrace)
# selenium.common.exceptions.ElementNotVisibleException: Message: Element is not currently visible and so may not be interacted with
# Stacktrace:
#     at fxdriver.preconditions.visible (file:///C:/Users/bzalakos/AppData/Local/Temp/tmp013gdefh/extensions/fxdriver@googlecode.com/components/command-processor.js:10092)
#     at DelayedCommand.prototype.checkPreconditions_ (file:///C:/Users/bzalakos/AppData/Local/Temp/tmp013gdefh/extensions/fxdriver@googlecode.com/components/command-processor.js:12644)
#     at DelayedCommand.prototype.executeInternal_/h (file:///C:/Users/bzalakos/AppData/Local/Temp/tmp013gdefh/extensions/fxdriver@googlecode.com/components/command-processor.js:12661)
#     at fxdriver.Timer.prototype.setTimeout/<.notify (file:///C:/Users/bzalakos/AppData/Local/Temp/tmp013gdefh/extensions/fxdriver@googlecode.com/components/command-processor.js:625)
not ok 12 - Result of: test_Footer (__main__.REGR) - Checks the content of the Footer.
# Traceback (most recent call last):
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 58, in testPartExecutor
#     yield
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 600, in run
#     testMethod()
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 227, in test_Footer
#     sofi('a[href="https://www.facebook.com/SeeAustralia"]')
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 68, in reul
#     elle = fiel(*parg, **karg)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webelement.py", line 282, in find_element_by_css_selector
#     return self.find_element(by=By.CSS_SELECTOR, value=css_selector)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webelement.py", line 477, in find_element
#     {"using": by, "value": value})['value']
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webelement.py", line 457, in _execute
#     return self._parent.execute(command, params)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 233, in execute
#     self.error_handler.check_response(response)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 194, in check_response
#     raise exception_class(message, screen, stacktrace)
# selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: {"method":"css selector","selector":"a[href=\"https://www.facebook.com/SeeAustralia\"]"}
# Stacktrace:
#     at FirefoxDriver.prototype.findElementInternal_ (file:///C:/Users/bzalakos/AppData/Local/Temp/tmpt9s97uot/extensions/fxdriver@googlecode.com/components/driver-component.js:10770)
#     at fxdriver.Timer.prototype.setTimeout/<.notify (file:///C:/Users/bzalakos/AppData/Local/Temp/tmpt9s97uot/extensions/fxdriver@googlecode.com/components/driver-component.js:625)
not ok 13 - Result of: test_Forgotten_Password (__main__.REGR) - 
# Traceback (most recent call last):
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 58, in testPartExecutor
#     yield
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 600, in run
#     testMethod()
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 599, in test_Forgotten_Password
#     verf('.fancybox-skin')
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 142, in assertElementPresent
#     waitfor.visibility_of(self.driver.find_element_by_css_selector(selector))))
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 434, in find_element_by_css_selector
#     return self.find_element(by=By.CSS_SELECTOR, value=css_selector)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 744, in find_element
#     {'using': by, 'value': value})['value']
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 233, in execute
#     self.error_handler.check_response(response)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 194, in check_response
#     raise exception_class(message, screen, stacktrace)
# selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: {"method":"css selector","selector":".fancybox-skin"}
# Stacktrace:
#     at FirefoxDriver.prototype.findElementInternal_ (file:///C:/Users/bzalakos/AppData/Local/Temp/tmpgh6ep_a_/extensions/fxdriver@googlecode.com/components/driver-component.js:10770)
#     at fxdriver.Timer.prototype.setTimeout/<.notify (file:///C:/Users/bzalakos/AppData/Local/Temp/tmpgh6ep_a_/extensions/fxdriver@googlecode.com/components/driver-component.js:625)
not ok 14 - Result of: test_Forgotten_Username (__main__.REGR) - 
# Traceback (most recent call last):
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 58, in testPartExecutor
#     yield
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 600, in run
#     testMethod()
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 585, in test_Forgotten_Username
#     verf('.fancybox-skin')
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 142, in assertElementPresent
#     waitfor.visibility_of(self.driver.find_element_by_css_selector(selector))))
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 434, in find_element_by_css_selector
#     return self.find_element(by=By.CSS_SELECTOR, value=css_selector)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 744, in find_element
#     {'using': by, 'value': value})['value']
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 233, in execute
#     self.error_handler.check_response(response)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 194, in check_response
#     raise exception_class(message, screen, stacktrace)
# selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: {"method":"css selector","selector":".fancybox-skin"}
# Stacktrace:
#     at FirefoxDriver.prototype.findElementInternal_ (file:///C:/Users/bzalakos/AppData/Local/Temp/tmpi2fz1m8_/extensions/fxdriver@googlecode.com/components/driver-component.js:10770)
#     at fxdriver.Timer.prototype.setTimeout/<.notify (file:///C:/Users/bzalakos/AppData/Local/Temp/tmpi2fz1m8_/extensions/fxdriver@googlecode.com/components/driver-component.js:625)
not ok 15 - Result of: test_Homepage (__main__.REGR) - Tests the Welcome Page content.
# Traceback (most recent call last):
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 58, in testPartExecutor
#     yield
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 600, in run
#     testMethod()
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 162, in test_Homepage
#     verf('#hero-player')
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 142, in assertElementPresent
#     waitfor.visibility_of(self.driver.find_element_by_css_selector(selector))))
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 434, in find_element_by_css_selector
#     return self.find_element(by=By.CSS_SELECTOR, value=css_selector)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 744, in find_element
#     {'using': by, 'value': value})['value']
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 233, in execute
#     self.error_handler.check_response(response)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 194, in check_response
#     raise exception_class(message, screen, stacktrace)
# selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: {"method":"css selector","selector":"#hero-player"}
# Stacktrace:
#     at FirefoxDriver.prototype.findElementInternal_ (file:///C:/Users/bzalakos/AppData/Local/Temp/tmpksc09tia/extensions/fxdriver@googlecode.com/components/driver-component.js:10770)
#     at fxdriver.Timer.prototype.setTimeout/<.notify (file:///C:/Users/bzalakos/AppData/Local/Temp/tmpksc09tia/extensions/fxdriver@googlecode.com/components/driver-component.js:625)
not ok 16 - Result of: test_Interactive_Map (__main__.REGR) - Checks the Interactive Map page.
# Traceback (most recent call last):
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 58, in testPartExecutor
#     yield
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 600, in run
#     testMethod()
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 416, in test_Interactive_Map
#     verf('#map-menu > #cities')
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 142, in assertElementPresent
#     waitfor.visibility_of(self.driver.find_element_by_css_selector(selector))))
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 434, in find_element_by_css_selector
#     return self.find_element(by=By.CSS_SELECTOR, value=css_selector)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 744, in find_element
#     {'using': by, 'value': value})['value']
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 233, in execute
#     self.error_handler.check_response(response)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 194, in check_response
#     raise exception_class(message, screen, stacktrace)
# selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: {"method":"css selector","selector":"#map-menu > #cities"}
# Stacktrace:
#     at FirefoxDriver.prototype.findElementInternal_ (file:///C:/Users/bzalakos/AppData/Local/Temp/tmps7yeys3y/extensions/fxdriver@googlecode.com/components/driver-component.js:10770)
#     at fxdriver.Timer.prototype.setTimeout/<.notify (file:///C:/Users/bzalakos/AppData/Local/Temp/tmps7yeys3y/extensions/fxdriver@googlecode.com/components/driver-component.js:625)
not ok 17 - Result of: test_Login (__main__.REGR) - 
# Traceback (most recent call last):
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 58, in testPartExecutor
#     yield
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 600, in run
#     testMethod()
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 572, in test_Login
#     self.assertFalse(nelec('#nav-main-panel-5'))
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 671, in assertFalse
#     raise self.failureException(msg)
# AssertionError: True is not false
not ok 18 - Result of: test_My_Profile (__main__.REGR) - 
# Traceback (most recent call last):
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 58, in testPartExecutor
#     yield
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 600, in run
#     testMethod()
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 730, in test_My_Profile
#     self.assertEqual(elec('[name="lname"]').get_attribute('value'), lno)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 820, in assertEqual
#     assertion_func(first, second, msg=msg)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 1193, in assertMultiLineEqual
#     self.fail(self._formatMessage(msg, standardMsg))
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 665, in fail
#     raise self.failureException(msg)
# AssertionError: 'TEST' != 'csstransitions TEST draganddrop TEST webs[39 chars]umns'
# - TEST
# + csstransitions TEST draganddrop TEST websockets TEST csstransforms3d TEST csscolumns
#
ok 19 - Result of: test_Navigation (__main__.REGR) - Checks that the contents of the Signed Out Nav Menu are correct.
not ok 20 - Result of: test_Premier_Badge (__main__.REGR) - 
# Traceback (most recent call last):
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 58, in testPartExecutor
#     yield
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 600, in run
#     testMethod()
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 1034, in test_Premier_Badge
#     verf('#profile-form > div:nth-child(2) a > img[src*="achievements/3.png"]')
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 142, in assertElementPresent
#     waitfor.visibility_of(self.driver.find_element_by_css_selector(selector))))
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 434, in find_element_by_css_selector
#     return self.find_element(by=By.CSS_SELECTOR, value=css_selector)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 744, in find_element
#     {'using': by, 'value': value})['value']
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 233, in execute
#     self.error_handler.check_response(response)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 194, in check_response
#     raise exception_class(message, screen, stacktrace)
# selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: {"method":"css selector","selector":"#profile-form > div:nth-child(2) a > img[src*=\"achievements/3.png\"]"}
# Stacktrace:
#     at FirefoxDriver.prototype.findElementInternal_ (file:///C:/Users/bzalakos/AppData/Local/Temp/tmp52znafm9/extensions/fxdriver@googlecode.com/components/driver-component.js:10770)
#     at fxdriver.Timer.prototype.setTimeout/<.notify (file:///C:/Users/bzalakos/AppData/Local/Temp/tmp52znafm9/extensions/fxdriver@googlecode.com/components/driver-component.js:625)
not ok 21 - Result of: test_Registration (__main__.REGR) - Checks the Registration process. Slight use of witchcraft.
# Traceback (most recent call last):
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 58, in testPartExecutor
#     yield
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 600, in run
#     testMethod()
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 502, in test_Registration
#     Select(elec('#country_id')).select_by_value(lcod.upper())
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 68, in reul
#     elle = fiel(*parg, **karg)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 434, in find_element_by_css_selector
#     return self.find_element(by=By.CSS_SELECTOR, value=css_selector)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 744, in find_element
#     {'using': by, 'value': value})['value']
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 233, in execute
#     self.error_handler.check_response(response)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 194, in check_response
#     raise exception_class(message, screen, stacktrace)
# selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: {"method":"css selector","selector":"#country_id"}
# Stacktrace:
#     at FirefoxDriver.prototype.findElementInternal_ (file:///C:/Users/bzalakos/AppData/Local/Temp/tmpqn1r05st/extensions/fxdriver@googlecode.com/components/driver-component.js:10770)
#     at fxdriver.Timer.prototype.setTimeout/<.notify (file:///C:/Users/bzalakos/AppData/Local/Temp/tmpqn1r05st/extensions/fxdriver@googlecode.com/components/driver-component.js:625)
not ok 22 - Result of: test_Sitemap (__main__.REGR) - Checks the Sitemap page links.
# Traceback (most recent call last):
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 58, in testPartExecutor
#     yield
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 600, in run
#     testMethod()
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 265, in test_Sitemap
#     self.assertTrue(moras.issubset(simas))
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 677, in assertTrue
#     raise self.failureException(msg)
# AssertionError: False is not true
not ok 23 - Result of: test_Splash_Page (__main__.REGR) - Tests the Splash Page.
# Traceback (most recent call last):
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 58, in testPartExecutor
#     yield
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 600, in run
#     testMethod()
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 149, in test_Splash_Page
#     op = elec('.dropdown-select-language1')
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 68, in reul
#     elle = fiel(*parg, **karg)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 434, in find_element_by_css_selector
#     return self.find_element(by=By.CSS_SELECTOR, value=css_selector)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 744, in find_element
#     {'using': by, 'value': value})['value']
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 233, in execute
#     self.error_handler.check_response(response)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 194, in check_response
#     raise exception_class(message, screen, stacktrace)
# selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: {"method":"css selector","selector":".dropdown-select-language1"}
# Stacktrace:
#     at FirefoxDriver.prototype.findElementInternal_ (file:///C:/Users/bzalakos/AppData/Local/Temp/tmpxqaqyyuh/extensions/fxdriver@googlecode.com/components/driver-component.js:10770)
#     at fxdriver.Timer.prototype.setTimeout/<.notify (file:///C:/Users/bzalakos/AppData/Local/Temp/tmpxqaqyyuh/extensions/fxdriver@googlecode.com/components/driver-component.js:625)
not ok 24 - Result of: test_Training_Summary (__main__.REGR) - 
# Traceback (most recent call last):
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 58, in testPartExecutor
#     yield
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 600, in run
#     testMethod()
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 751, in test_Training_Summary
#     elec('[name="Optional"]'), 'TrainingModule:Optional/Niche')
#   File "C:\Users\bzalakos\Desktop\Snippets-1\py\hiptest\selene.py", line 68, in reul
#     elle = fiel(*parg, **karg)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 434, in find_element_by_css_selector
#     return self.find_element(by=By.CSS_SELECTOR, value=css_selector)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 744, in find_element
#     {'using': by, 'value': value})['value']
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 233, in execute
#     self.error_handler.check_response(response)
#   File "C:\Users\bzalakos\AppData\Local\Programs\Python\Python35-32\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 194, in check_response
#     raise exception_class(message, screen, stacktrace)
# selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: {"method":"css selector","selector":"[name=\"Optional\"]"}
# Stacktrace:
#     at FirefoxDriver.prototype.findElementInternal_ (file:///C:/Users/bzalakos/AppData/Local/Temp/tmpmwszeyah/extensions/fxdriver@googlecode.com/components/driver-component.js:10770)
#     at fxdriver.Timer.prototype.setTimeout/<.notify (file:///C:/Users/bzalakos/AppData/Local/Temp/tmpmwszeyah/extensions/fxdriver@googlecode.com/components/driver-component.js:625)
ok 25 - Result of: test_Travel_Club (__main__.REGR) - 
1..25
